src/cc_catalog_airflow/dags/provider_api_scripts/common/storage/image.py

```diff
@@ -11,66 +11,92 @@
-        name='foreign_identifier',  required=False, size=3000, truncate=False
+        name='foreign_identifier', required=False, size=3000, truncate=False
     ),
     columns.URLColumn(
-        name='foreign_landing_url', required=True,  size=1000
+        name='foreign_landing_url', required=True, size=1000
     ),
     columns.URLColumn(
         # `url` in DB
-        name='image_url',           required=True,  size=3000
+        name='image_url', required=True, size=3000
     ),
     columns.URLColumn(
         # `thumbnail` in DB
-        name='thumbnail_url',       required=False, size=3000
+        name='thumbnail_url', required=False, size=3000
     ),
     columns.IntegerColumn(
-        name='width',               required=False
+        name='width', required=False
     ),
     columns.IntegerColumn(
-        name='height',              required=False
+        name='height', required=False
     ),
     columns.IntegerColumn(
-        name='filesize',            required=False
+        name='filesize', required=False
     ),
     columns.StringColumn(
-        name='license_',            required=True,  size=50,   truncate=False
+        name='license_', required=True, size=50, truncate=False
     ),
     columns.StringColumn(
-        name='license_version',     required=True,  size=25,   truncate=False
+        name='license_version', required=True, size=25, truncate=False
     ),
     columns.StringColumn(
-        name='creator',             required=False, size=2000, truncate=True
+        name='creator', required=False, size=2000, truncate=True
     ),
     columns.URLColumn(
-        name='creator_url',         required=False, size=2000
+        name='creator_url', required=False, size=2000
     ),
     columns.StringColumn(
-        name='title',               required=False, size=5000, truncate=True
+        name='title', required=False, size=5000, truncate=True
     ),
     columns.JSONColumn(
-        name='meta_data',           required=False
+        name='meta_data', required=False
     ),
     columns.JSONColumn(
-        name='tags',                required=False
+        name='tags', required=False
     ),
     columns.BooleanColumn(
-        name='watermarked',         required=False
+        name='watermarked', required=False
     ),
     columns.StringColumn(
-        name='provider',            required=False, size=80,   truncate=False
+        name='provider', required=False, size=80, truncate=False
     ),
     columns.StringColumn(
-        name='source',              required=False, size=80,   truncate=False
+        name='source', required=False, size=80, truncate=False
     )
 ]
 
-
 _Image = namedtuple(
     '_Image',
     [c.NAME for c in _IMAGE_TSV_COLUMNS]
 )
 
+# Filter out tags that exactly match these terms. All terms should be
+# lowercase.
+TAG_BLACKLIST = {
+    'no person',
+    'squareformat',
+    'uploaded:by=flickrmobile',
+    'uploaded:by=instagram',
+    'flickriosapp:filter=flamingo'
```

Comment:
These will be filtered out by the `TAG_CONTAINS_BLACKLIST` step, no?

Reply:
Except for the 'squareformat' and 'no person', yes all others are covered in the <code>TAG_CONTAINS_BLACKLIST</code>. I will remove them from here.